### PR TITLE
Rehome QueryModelResource to pulpcore.plugin.importexport

### PR DESCRIPTION
**Pull request by ggainey**
fixes #6514

### CODE REVIEW

File: CHANGES/6514.removal [hunk 1]
```diff
@@ -0,0 +1 @@
+Rehomed QueryModelResource to pulpcore.plugin.importexport.
```
Comment (daviddavis) [inline CHANGES/6514.removal:1]: I think this should go in the plugin api changelog (`CHANGES/plugin_api`).

File: docs/plugins/plugin-writer/concepts/subclassing/import-export.rst [hunk 2]
```diff
@@ -40,7 +40,7 @@ modelresource.py
 
 A simple ``modelresource.py`` module might look like this::
 
-    from pulpcore.app.modelresource import QueryModelResource
+    from pulpcore.plugin.importexport import QueryModelResource
```
Comment (daviddavis) [inline docs/plugins/plugin-writer/concepts/subclassing/import-export.rst:43]: Good catch :+1:

File: CHANGES/6514.removal (hunk 1 above)
Comment (ggainey) [reply CHANGES/6514.removal]: concur, moved

File: docs/plugins/plugin-writer/concepts/subclassing/import-export.rst (hunk 2 above)
Comment (ggainey) [reply docs/plugins/plugin-writer/concepts/subclassing/import-export.rst:44]: :)

**Pull request merged**